### DISCUSSION TITLE
How to render large number of children more efficiently in Scroll?

### DISCUSSION THREAD

**Discussion by flutterdevnp** [community/community#28066] [Mobile] [answered]
How to render large number of children more efficiently in Scroll?

**Comment by sthakrishnakumar** [comment, marked as answer]
Hello @flutterdevnp ,
                    Well while rendering the large number of children using Listview or Listview Builder may allocate large memory space which can affect in the app performance.
To overcome this, one can use Slivers for the efficiency of scrolling in flutter app.
You can get more from the official website as well. Link : https://docs.flutter.dev/development/ui/advanced/slivers

Hope this would help you at some extent. Thank You

**Reply by flutterdevnp** [reply]
Thank you @sthakrishnakumar for answer
